fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraVersioningIT.java

```diff
@@ -1021,9 +1021,15 @@ public void testDatetimeNegotiationLDPRv() throws Exception {
 
         final CloseableHttpClient customClient = createClient(true);
 
-        final String uri = createVersionedContainer(id);
+        createVersionedContainer(id);
+
+        final String version1Uri;
 
-        final String version1Uri = createMemento(uri);
+        try (final CloseableDataset dataset = getDataset(getObjMethod(id + "/" + FCR_VERSIONS))) {
```

Comment:
Given the number of times the following is implemented in this test class:
```
final String versionUri = createMemento(uri);
```
..it is conceivable that pulling this new logic into a new `private` method could be useful.